opendevin/events/action/commands.py

```diff
@@ -10,6 +10,14 @@
 class CmdRunAction(Action):
     command: str
     thought: str = ''
+    keep_prompt: bool = True
```

Comment:
Giving a "root@hostname:/workspace" provides additional info to the agent about its workspace, which I'd say is an improvement to our older ServerRuntime version.
This also allow us to support #2401 better, because we can also display python virtualenv `(venv)` in that prompt.